src/static/helpers/slasHelper.ts

```diff
@@ -306,9 +307,15 @@ export function logout(
     clientId: string;
     siteId: string;
   }>,
-  parameters: {refreshToken: string}
+  parameters: {
+    accessToken: string;
```

Comment:
This violates the established pattern of `parameters` being "things in the URL". I don't have a better solution, though.